conf/cortx_rgw.conf

```diff
@@ -11,15 +11,21 @@
     rgw thread pool size = 10
     rgw max concurrent request = 10
     rgw init timeout = 300
-    rgw gc max objs = 32
-    rgw gc obj min wait = 1800
-    rgw gc processor max time = 3600
-    rgw gc processor period = 3600
     rgw data path = /var/lib/ceph/radosgw/<cluster-id>   # Cluster id of cortx cluster
 
     # Enabling cache causes data retrieval inconsistencies for get-object, head-object etc. ref <CORTX-31109>
     rgw cache enabled = false
 
+    # GC configuration paraneters
+    rgw enable gc threads = true
+    rgw gc obj min wait = 7200        # 2hours
+    rgw gc processor period = 3600    # 1hour
+    rgw gc max concurrent io = 1
+    rgw gc max trim chunk = 256
+    rgw gc max objs = 64
+    rgw gc processor max time = 3600  # 1hour TBD
```

Comment:
@sachinpunadikar , @jjxsg  this last parameter was not part of my JIRA but based on tonight's discussion with basav.
Could you please confirm if we need to keep it or remove it.

Reply:
lets keep it right now